examples/cpp/core/capped_allocator/capped_allocator.bond

```diff
@@ -0,0 +1,13 @@
+namespace examples.capped_allocator
+
+struct Strings
+{
+	0: vector<string> strings;
+}
+
+struct Struct
+{
+    0: vector<int32>	ints;
```

Comment:
Minor: can we fix the odd indentation (mix or tabs and spaces?)

Reply:
Fixed.